src/status_im/subs/wallet/wallet.cljs

```diff
@@ -198,3 +198,11 @@
  (fn [[current-viewing-account network-details]]
    (let [network-preferences-names (:network-preferences-names current-viewing-account)]
      (filter #(contains? network-preferences-names (:network-name %)) network-details))))
+
+(rf/reg-sub
+ :wallet/accounts-with-customization-color
+ :<- [:wallet/accounts]
+ (fn [accounts]
+   (mapv (fn [{:keys [color] :as account}]
+           (assoc account :customization-color color))
```

Comment:
This subscription just duplicates the value of `:customization-color` in each `account` map. I don't see much value in this processing taking place as a subscription. Why can't the view use the `:color` key directly?

Reply:
I see, thanks.